remote_attestation/rust/src/crypto.rs

```diff
@@ -161,6 +160,12 @@ impl AeadEncryptor {
     }
 }
 
+/// Convenience struct for representing generated session keys.
+pub struct SessionKeys {
```

Comment:
How is this different from AeadEncryptor above?

Reply:
Returned back to `(encryption_key, decryption_key)`.
It doesn't use `AeadEncryptor` because `derive_session_keys` is used in tests for checking that server decryption key is equal to client encryption key etc.
And the function is used only once in the actual code.

Reply:
Please don't use a pair of two identical types, we will definitely mix them up at some point. Either keep the struct, or make the fields of the existing struct public, or something else. Or just clarify how the two structs are different, which is what I was asking (and you answered here).

Reply:
It's still looks scary to make private keys public (even `pub(crate)`).
I think I'll just make 2 structs: `EncryptionKey([u8; ...]` and `DecryptionKey([u8; ...])`.